Makefile

```diff
@@ -6,10 +6,11 @@
 TOP = .
 include $(TOP)/build.mk
 
-# TODO bring back demo
-# SUB_DIRS = utils ercc ecc_enclave ecc tlcc_enclave tlcc examples integration demo # docs
-SUB_DIRS = common utils ercc ecc_enclave ecc tlcc_enclave tlcc plugins fabric examples integration
-FPC_SDK = utils/fabric ecc_enclave ecc
+SUB_DIRS = common utils ercc ecc_enclave ecc tlcc_enclave tlcc plugins fabric examples integration demo # docs
+
+FPC_SDK = utils/fabric common ecc_enclave ecc
+FPC_PEER = plugins tlcc_enclave tlcc fabric ecc_enclave ecc # ecc & ecc_enclave we need if we run on host in external builder, not via docker
+FPC_PEER_CLI = utils/fabric
```

Comment:
I was confused by this, until I realized that that is not the cli, nor there is anything to build about that.
Suggestion, for both this is the target: `FPC_UTILS` or similar.

Reply:
it the list of build dependency directories to run peer cli.   It matches the pattern we had for FPC_SDK and FPC_PEER.   Arguably the pattern doesn't capture that it is a list of directories and that they form dependencies.    It would be more verbose but maybe more clear to change all three to `..._DEPEND_DIRS` .   Thoughts? A better name?

Reply:
> it the list of build dependency directories to run peer cli.

so, as I said, it's not the CLI.

> It matches the pattern we had for FPC_SDK and FPC_PEER. 

though FPC_PEER contains fabric, which will eventually build the peer.

> Arguably the pattern doesn't capture that it is a list of directories and that they form dependencies. It would be more verbose but maybe more clear to change all three to `..._DEPEND_DIRS` . Thoughts? A better name?

Whatever is less misleading, `..._DEPEND_DIRS`, `..._DEPS`, ...

Reply:
> > It matches the pattern we had for FPC_SDK and FPC_PEER.
> 
> though FPC_PEER contains fabric, which will eventually build the peer.

I guess you are correct that our fpc peer (wrapper) also depends on peer and so strictly speaking FPC_PEER_CLI should also include $FPC_PEER.  That said, from a cli perspective we actually do not really need FPC-modifed peer binary and in some docker script scenario it actually might currently run on the downloaded rather than self-built binaries ...